### DISCUSSION TITLE
Not working w/ LazyVim (the config + lazynvim package manager)

### DISCUSSION THREAD

**Discussion by vbfischer** [abecodes/tabout.nvim#45] [Q&A] [answered]
I'm pretty sure this is something stupid I'm doing as it was working fine in another setup I was working on. I switched to Lazy.Vim and added the following to my `plugins` folder:

```
return {
  "abecodes/tabout.nvim",
  event = "VeryLazy",
  dependencies = {
    "nvim-treesitter/nvim-treesitter",
    "hrsh7th/nvim-cmp",
  },
  config = true,
}
```

Viewing the Lazy setup it shows it loaded
![CleanShot 2023-02-24 at 09 00 34](https://user-images.githubusercontent.com/138106/221197154-1e9db19f-0b70-4601-9602-9bd1a60f5c2f.png)

The command `:Tabout` exists but it returns the following warning:

![CleanShot 2023-02-24 at 09 01 36](https://user-images.githubusercontent.com/138106/221197428-853d771d-2210-4769-9c89-be08ec389b55.png)

**Comment by vbfischer** [comment, marked as answer]
I figured it out. LuaSnip was using the `<tab>` as a key 

```
 {
    "L3MON4D3/LuaSnip",
    dependencies = {
      "rafamadriz/friendly-snippets",
      config = function()
        require("luasnip.loaders.from_vscode").lazy_load()
      end,
    },
    opts = {
      history = true,
      delete_check_events = "TextChanged",
    },
    -- stylua: ignore
    keys = {
      {
        "<tab>",
        function()
          return require("luasnip").jumpable(1) and "<Plug>luasnip-jump-next" or "<tab>"
        end,
        expr = true, silent = true, mode = "i",
      },
      { "<tab>", function() require("luasnip").jump(1) end, mode = "s" },
      { "<s-tab>", function() require("luasnip").jump(-1) end, mode = { "i", "s" } },
    },
  },
```

so just unsetting it worked

```
return {
  "L3MON4D3/LuaSnip",
  keys = function()
    return {}
  end,
}
```

source https://www.lazyvim.org/configuration/recipes#supertab

**Comment by abecodes** [comment]
Hi @vbfischer
Thanks for sharing and closing 🥇 
Glad that fixed your issue, but it is a little bit weird. 
Actually the warning is telling you that you are using `:tabout` in a binding when it should be `<Plug>(Tabout)`.

**Reply by vbfischer** [reply]
Probably because I had done `:Tabout` in nvim. It will still give that warning if I type that. but the <tab> functionality works great now without specifying any bindings. Now just researching how to do effective code folding in a React projectf...
